"""
Custom exceptions
"""


class AuthError(Exception):
    """Problem with auth"""
